pkg/setting/setting.go

```diff
@@ -429,6 +429,11 @@ type Cfg struct {
 	UnifiedAlerting UnifiedAlertingSettings
 }
 
+// IsValidatedQueriesEnabled
+func (cfg Cfg) IsValidatedQueriesEnabled() bool {
+	return cfg.FeatureToggles["validatedQueries"]
```

Comment:
now that #44443  I think you need to check `hs.Features.IsEnabled()` instead.

Reply:
Thanks @papagian! The featuremgmt package was released before we authored this. Very exciting : )